services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/promptsForm.tsx

```diff
@@ -16,38 +16,84 @@ interface PromptsFormProps {
-  function handleSetButPrompt (e: InputEvent) { handleSetPrompt(e.target.value, C.BUT, 'text') }
-  function handleSetSoPrompt (e: InputEvent) { handleSetPrompt(e.target.value, C.SO, 'text') }
+  function handleSetBecausePromptText (e: InputEvent) { handleSetPrompt(e.target.value, C.BECAUSE, C.TEXT) }
+  function handleSetBecausePromptFirstStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.BECAUSE, C.FIRST_STRONG_EXAMPLE) }
+  function handleSetBecausePromptSecondStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.BECAUSE, C.SECOND_STRONG_EXAMPLE) }
+  function handleSetButPromptText (e: InputEvent) { handleSetPrompt(e.target.value, C.BUT, C.TEXT) }
+  function handleSetButPromptFirstStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.BUT, C.FIRST_STRONG_EXAMPLE) }
+  function handleSetButPromptSecondStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.BUT, C.SECOND_STRONG_EXAMPLE) }
+  function handleSetSoPromptText (e: InputEvent) { handleSetPrompt(e.target.value, C.SO, C.TEXT) }
+  function handleSetSoPromptFirstStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.SO, C.FIRST_STRONG_EXAMPLE) }
+  function handleSetSoPromptSecondStrongExample (e: InputEvent) { handleSetPrompt(e.target.value, C.SO, C.SECOND_STRONG_EXAMPLE) }
 
   return(
     <React.Fragment>
+      <div className="info-section">
+        <img alt={informationIcon.alt} src={informationIcon.src} />
+        <p>You do not need to retype the stem for exemplars.</p>
+      </div>
       <section className="prompt-section">
         <Input
           className="because-input"
           error={errors[C.BECAUSE_STEM]}
-          handleChange={handleSetBecausePrompt}
+          handleChange={handleSetBecausePromptText}
           label="Because Stem"
           value={activityBecausePrompt.text}
         />
+        <Input
+          className="because-example"
+          handleChange={handleSetBecausePromptFirstStrongExample}
+          label="Because First Exemplar"
+          value={activityBecausePrompt.first_strong_example}
+        />
+        <Input
+          className="because-example"
+          handleChange={handleSetBecausePromptSecondStrongExample}
+          label="Because Second Exemplar"
+          value={activityBecausePrompt.second_strong_example}
+        />
       </section>
       <section className="prompt-section">
         <Input
           className="but-input"
           error={errors[C.BUT_STEM]}
-          handleChange={handleSetButPrompt}
+          handleChange={handleSetButPromptText}
           label="But Stem"
           value={activityButPrompt.text}
         />
+        <Input
+          className="but-example"
+          handleChange={handleSetButPromptFirstStrongExample}
+          label="But First Exemplar"
+          value={activityButPrompt.first_strong_example}
+        />
+        <Input
+          className="But-example"
```

Comment:
Not that it matters, but is the `But` here accidentally capitalized?

Reply:
it is, I'll change it